providers/implementations/ciphers/ciphercommon.c

```diff
@@ -550,6 +563,9 @@ int cipher_generic_set_ctx_params(void *vctx, const OSSL_PARAM params[])
 int cipher_generic_initiv(PROV_CIPHER_CTX *ctx, const unsigned char *iv,
                           size_t ivlen)
 {
+    if (!ossl_prov_is_running())
```

Comment:
not sure this one is needed..

Reply:
It's safely guarded everywhere except for`rc2_set_ctx_params()`.  The new/dup/init will have caught this for new algorithms and the final catches it for existing and it isn't doing anything cryptographic, so consider it gone.